Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+P3A.swift

```diff
@@ -93,6 +95,75 @@ extension BrowserViewController {
     
     usage.recordReturningUsageMetric()
   }
+  
+  func recordAccessibilityDisplayZoomEnabledP3A() {
+    // Accessibility Q1 New P3A iOS - Do you have iOS display zoom enabled?
+    let isDisplayZoomEnabled = UIScreen.main.scale < UIScreen.main.nativeScale
+    
+    UmaHistogramBoolean("Brave.Accessibility.DisplayZoomEnabled", isDisplayZoomEnabled)
+  }
+  
+  func recordAccessibilityDocumentsDirectorySizeP3A() {
+    // Accessibility Q4 New P3A iOS - Documents directory size
+    func fetchDocumentsAndDataSize() -> Int? {
+      let fileManager = FileManager.default
+      
+      guard let documentsDirectory = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first else {
+        return nil
+      }
+
+      do {
+        if let documentsDirectorySize = try fileManager.directorySize(at: documentsDirectory){
+          return Int(documentsDirectorySize / 1024 / 1024)
+        } else {
+          return nil
+        }
+      } catch {
+        return nil
+      }
+    }
+    
+    let buckets: [Bucket] = [
+      .r(0...50),
+      .r(50...200),
+      .r(200...500),
+      .r(500...1000),
+      .r(1000...),
+    ]
+    
+    if let documentsSize = fetchDocumentsAndDataSize() {
+      UmaHistogramRecordValueToBucket("Brave.Accessibility.DocumentDirectorySize", buckets: buckets, value: documentsSize)
+    }
+  }
+
+  func recordGeneralBottomBarLocationP3A() {
+    enum Answer: Int, CaseIterable {
+      case top = 0
+      case bottom = 1
+    }
+    
+    // General Q1 New P3A iOS - Bottom Bar Location
+    let answer: Answer = Preferences.General.isUsingBottomBar.value ? .bottom : .top
```

Comment:
100 percent valid question and we must not count iPad users in this category, it will be misleading. Fixed at https://github.com/brave/brave-ios/pull/7100/commits/2696620bb9c7ae416c929338baff6ea432836c01